enterprise.html

```diff
@@ -278,7 +278,7 @@
 };</code
 						></pre>
           <img
-            src="./Screen_ee.png"
+            src="./screen_ee.png"
```

Comment:
The upper case is inconsistent with the other image. Also since only Linux is case-sensitive, this creates problems on macOS and Windows.